src/test/java/de/tum/in/www1/artemis/localvcci/LocalVCSshIntegrationTest.java

```diff
@@ -142,7 +142,6 @@ void testConnectOverSshAndReceivePack() {
         }
         catch (NullPointerException e) {
             assertThat(e).isInstanceOf(NullPointerException.class);
```

Comment:
A object of type `NullPointerException` is always of type `NullPointerException`, this assertion is useless.
```suggestion
```
Was the false is true assertion meant to fail the test if an exception occurs? If yes, you could just remove the try-catch-block.